cg/meta/workflow/analysis.py

```diff
@@ -483,11 +491,16 @@ def ensure_flow_cells_on_disk(self, case_id: str) -> None:
             self.status_db.request_flow_cells_for_case(case_id)
 
     def is_case_ready_for_analysis(self, case_id: str) -> bool:
```

Comment:
Lots of logic - a doc str would not hurt.

Reply:
Broke it up into two smaller methods!